pages/common/gimp.md

```diff
@@ -1,25 +1,25 @@
 # gimp
 
-> An image manipulation and paint program.
+> GNU image manipulation program.
 > More information: <https://docs.gimp.org/en/gimp-fire-up.html#gimp-concepts-running-command-line>.
 
 - Launch GIMP:
 
 `gimp`
 
-- Launch but don't show the splash screen:
+- Launch, but don't show the splash screen:
```

Comment:
When possible, I'd suggest avoiding context-sensitive example descriptions (i.e. relying on the sentence structure of the previous example's description):

```suggestion
- Launch GIMP without showing the splash screen:
```

Reply:
Hmm...we do that in a lot of pages though.  I actually rely on it if the previous description is lengthy ;)

Reply:
Exactly, it makes sense as a trade-off to balance the length of the description. In this case, it doesn't seem to be necessary, so there's only the downside of the reliance on another description.

Reply:
Ah, I see what you're saying, then.  Yes, that does make _perfect_ sense.

Reply:
Great idea! Let's do this.